### PR TITLE
Remove experimental flag and OT from PWA widgets doc

### LINKED ISSUE #2749: Where is "To enable widget support in Microsoft Edge for users of your production PWA, enroll your site in the Web App Widgets origin trial."  ?
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1a0e6a65-5e0a-55cc-d3c8-b4fdd37a2d2b
* Version Independent ID: 1a0e6a65-5e0a-55cc-d3c8-b4fdd37a2d2b
* Content: [Build PWA-driven widgets - Microsoft Edge Development](https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets#enable-widgets-for-your-users)
* Content Source: [microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md](https://github.com/MicrosoftDocs/edge-developer/blob/main/microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md)
* Product: **microsoft-edge**
* Technology: **pwa**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

AB#45803802

### PR DESCRIPTION
**Pull request by captainbrosset**
The PWA Widgets feature shipped with Edge 115, and the origin trial does not exist anymore.
This PR updates the docs accordingly.

Fixes #2749.

Previews:
* GitHub: https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/widget-ot/microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md
* Deploy: https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets?branch=pr-en-us-2753

AB#45821205

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md [hunk 1]
```diff
@@ -23,36 +19,6 @@ On Windows 11, widgets appear in the widgets board which you open from the left
...
-
-To enable developing widgets on your local machine:
-
-1. Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
-
-1. In Microsoft Edge, go to `edge://flags`.
-
-1. In the **Search flags** text box, type **Web App widgets**, and then in the **Web App widgets** dropdown menu, select **Enabled**:
-
-   ![Enabling the Web App widgets flag in the edge://flags page](./widgets-images/widgets-flag.png)
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:41]: delete png files?
./widgets-images/widgets-flag.png
./widgets-images/experimental-features-flag.png

**Review by mikehoffms — APPROVED** [review]
Approved and asked if remove png files.  [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2753/files#r1283636334)

File: microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md [hunk 2]
```diff
@@ -23,36 +19,6 @@ On Windows 11, widgets appear in the widgets board which you open from the left
...
-<!-- ====================================================================== -->
-## Enable widgets in Microsoft Edge
-
-Widgets are experimentally supported starting with Microsoft Edge 108.
-
-#### Enable local development of widgets
-
-To enable developing widgets on your local machine:
-
-1. Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
```
Comment (amandabaker) [inline microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:35]: I believe installign WinAppSDK 1.2 is still required for local development, but the steps below are no longer necesary

Comment (mikehoffms) [reply microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:35]: i plan to create Chat thread for this PR, & plan to restore this line

Comment (mikehoffms) [reply microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:35]: Restored heading "Enable local development of widgets" promoted to h2.  
Restored the single step.
In Demo App section, restored/reworded step 1:
```
1. If not done yet, install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
```
@captainbrosset @amandabaker

File: microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:41]: deleted pngs

**Review by mikehoffms — APPROVED** [review]

**Review by amandabaker — APPROVED** [review]
LGTM.

**Pull request merged**